src/patternfly/components/Label/label-content.hbs

```diff
@@ -1,11 +1,13 @@
-<{{# if label-content--IsLink}}a{{else}}span{{/if}} class="pf-c-label__content{{#if label-content--modifier}} {{label-content--modifier}}{{/if}}"
+{{# if label--IsEditable}}
+  {{#> label-editable-text}}{{> @partial-block}}{{/label-editable-text}}
+{{else}}
+<{{# if label-content--IsLink}}a{{else if label--IsEditable}}button{{else}}span{{/if}} class="pf-c-label__content{{#if label-content--modifier}} {{label-content--modifier}}{{/if}}"
```

Comment:
looks like you could take out the `label--IsEditable` vars in this conditional block since if the var is set, this part isn't included.